build.sbt

```diff
@@ -1,13 +1,13 @@
 import sbtassembly.AssemblyPlugin.defaultUniversalScript
 
-lazy val kamuCoreManifests = RootProject(file("../kamu-core-manifests"))
+lazy val kamuCoreManifests = RootProject(file("kamu-core-manifests"))
```

Comment:
I think this needs a bit more reorganization. The structure with submodules I imagined (and seen in few other projects) is:
```
/kamu-cli
    src
    build.sbt
/kamu-core-manifests
    src
    build.sbt
/kamu-core-ingest-polling
    src
    build.sbt
/kamu-core-transform-streaming
    src
    build.sbt
/build.sbt
```
In other words we should put `kamu-cli` source code into subfolder with its own `build.sbt` and then have a `build.sbt` at project root that ties everything together.

Reply:
This would look like multiple packages within a single repository. There are repositories like that:
https://github.com/plredmond/GPipe-GLFW
https://github.com/portnov/libssh2-hs
But they don't use submodules. I would rather put submodules in a separate folder like here:
https://github.com/haskell/haskell-ide-engine (submodules)
https://gitlab.haskell.org/ghc/ghc (libraries)

Reply:
I like the idea of `libraries` folder. I was aiming for a standard multi-project layout of java/maven projects, but indeed none of them that I've seen used submodules, so having them alongside the other code might be confusing.

Reply:
Done